### DISCUSSION TITLE
Trouble installing the PCRE cask on OSX 10.8.5

### DISCUSSION THREAD

**Discussion by fstltna** [Homebrew/discussions#5158] [Casks] [answered]
### Output of `brew config`

```shell
My Mac cant access this website due to not having modern SSL certs so I can't paste
```

### Output of `brew doctor`

```shell
My Mac cant access this website due to not having modern SSL certs so I can't paste
```

### Description of issue

Trying to use:
brew install pcre
and it fails on one of the steps in the "make test" stage.

### Relevant casks

pcre

**Comment by kencu** [comment, marked as answer]
You will have great struggles trying to get the current homebrew working on MacOS 10.8.5. 

I am assuming 10.8.5 is as far as the machine can be updated.

The recommended upgrade path for you would be to install Ubuntu Linux on that hardware, then you could use the current homebrew on that should you choose to, and have access to much current software like Chrome, Firefox, QT, etc.

If you must or prefer to stay on MacOS you might try TigerBrew, https://github.com/mistydemeo/tigerbrew, although it is geared more towards supporting powerpc macs.

Your best bet would be MacPorts, that still actively supports 10.8.5. There will still be numerous headaches with that too, however.

For a working browser for your system, many find this one is quite usable:

https://github.com/blueboxd/chromium-legacy

**Reply by fstltna** [reply]
Thanks for the suggestions! I use ubuntu day to day but I am trying out running early versions of OSX as I don't like the more "modern" versions of the UI...
